### DISCUSSION TITLE
RPi 5 - Mitmweb Invalid mode specification: wireguard

### DISCUSSION THREAD

**Discussion by 641i130** [mitmproxy/mitmproxy#7140] [Q&A] [answered]
When trying to run this on a Raspberry Pi 5, I get this issue:

`/usr/bin/mitmweb: Invalid mode specification: wireguard`

Is this just not supported?

`mitmproxy/stable,stable,now 8.1.1-2 all [installed]`

**Comment by mhils** [comment, marked as answer]
You're using an outdated mitmproxy version.

**Comment by 641i130** [comment]
Installing via `pipx` fixed this!
